pom.xml

```diff
@@ -21,12 +21,12 @@
     <avro.version>1.7.7</avro.version>
     <!-- informative only, used in about template substitution -->
     <scala.version>2.10</scala.version>
-    <spark.version>1.5.2</spark.version>
+    <spark.version>1.6.1</spark.version>
```

Comment:
Similar to my comment upstream, I would just split this out into a separate commit.